tests/playwright/tests/columnQrCode.spec.ts

```diff
@@ -0,0 +1,103 @@
+        },
+      ];
+
+      // close 'Team & Auth' tab
+      await dashboard.closeTab({ title: 'Team & Auth' });
+
+      await dashboard.treeView.openTable({ title: 'City' });
+
+      await dashboard.grid.column.create({
+        title: 'QrCode1',
+        type: 'QrCode',
+        qrCodeValueColumnTitle: 'City',
+      });
+
+      await qrCodeColumnVerify('QrCode1', expectedQrCodeCellValues);
+
+      // Clicking on qr code in first row and expect it shows a
+      // popup with an enlarged version of the qr code
+      await dashboard.grid.cell.get({ columnHeader: 'QrCode1', index: 0 }).click();
+      const qrGridOverlay = dashboard.grid.qrCodeOverlay;
+      await qrGridOverlay.verifyQrValueLabel(expectedQrCodeCellValues[0].referencedValue);
+      await qrGridOverlay.clickCloseButton();
+
+      await dashboard.closeTab({ title: 'City' });
```

Comment:
Add test case for the case where the cell value(city name) is changed and verify that the QR code is changed with the correct value

Reply:
Extended the test (in the same test block - felt easier and probably speeds up the whole test suite since prep actions don't need to be done again for each test case, let me know though if you folks prefer more atomic test blocks)

Reply:
@spaudanjo We are not doing atomic test block, so the approach you took is the correct way

Reply:
great, thanks